src/models/network.js

```diff
@@ -73,10 +73,13 @@ Network.prototype.export = function() {
 		"hostname"
 	]);
 
-	network.join = _.map(
-		_.filter(this.channels, {type: Chan.Type.CHANNEL}),
-		"name"
-	).join(",");
+	network.channels = _
+		.filter(this.channels, {type: Chan.Type.CHANNEL})
```

Comment:
I know it's a copy from the above, but is it possible to use:

```js
this.channels.filter(function(channel) { return channel.type === Chan.Type.CHANNEL; });
```

? If so, let's do it? Please?